includes/class-wc-payment-token-wcpay-link.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Class WC_Payment_Token_WCPay_SEPA
+ *
+ * @package WooCommerce\Payments
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * WooCommerce Stripe Link Payment Token.
+ *
+ * Representation of a payment token for Link.
+ *
+ * @class    WC_Payment_Token_WCPay_Link
+ */
+class WC_Payment_Token_WCPay_Link extends WC_Payment_Token {
+
+	/**
+	 * Class Constant so other code can be unambiguous.
+	 *
+	 * @type string
+	 */
+	const TYPE = 'wcpay_link';
+
+	/**
+	 * The payment method type of this token.
+	 *
+	 * @var string
+	 */
+	protected $type = self::TYPE;
+
+	/**
+	 * Stores Link payment token data.
+	 *
+	 * @var array
+	 */
+	protected $extra_data = [
+		'email' => '',
+	];
+
+	/**
+	 * Get type to display to user.
```

Comment:
The description doesn't align with the method name. Is the description correct?

Reply:
added more context. thx